packages/db/postgres/docker/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.6'
 
 services:
   rest:
-    image: postgrest/postgrest:v8.0.0
```

Comment:
Is this required for the filtering to work?

If that's the case we need to make sure the version we're using on Heroku matches this one.

Reply:
It is, and has already been bumped on main.